### DISCUSSION TITLE
Unable to set wifi on raspberry pi 4 with Kdoren's jamulus

### DISCUSSION THREAD

**Discussion by jfrommer2001** [jamulussoftware/jamulus#1262] [Technical Support & Help] [answered]
No matter what I tried, wlan0 is unavailable on Kdoren's jamulus.
WiFI is working perfectly on the same hardware using an "official" raspberry pi 4 software.
Any clue??

**Comment by ann0see** [comment]
Probably it's disabled. Can you check the /boot/config.txt or commandline.txt file?

Just to be clear: Jamulus will not work correctly with WiFi.

**Comment by jfrommer2001** [comment]
Hi
Here is a copy were wifi is mentioned:
[all]
#dtoverlay=vcd-fkms-v3d
dyoverlay=disable-wifi

I will comment this line and will report

Thanks

**Reply by dcorson-ticino-com** [reply]
That is quite possible, but it is more important for you to get a cable.
You will not be happy with the audio quality using Jamulus with wifi on any computer.
Using a cable makes a big difference.

**Comment by jfrommer2001** [comment]
Hi
I am aware that jamulus will not work correctly with wifi
But, I intend to use the Pi for other tasks as well, thua I want to connect
it via wifi too.
I commented the [all] section in /boot/config.txt, the result of which was
that the internet connection was crossed saying : "no wireless interface
found"
(I guess that part of the problem now stems from the many changes I have
made trying to enable wifi connection.  As I am very poor with setting up
pi, I'll appreciate if you can help me  step by step ,  to set both wired
and wireless connection)
Thanks

**Reply by ann0see** [reply]
Did you try to install the wifi driver on your Pi? If you also use it for other tasks, you could also just use plain Raspberry Pi OS?

**Comment by jfrommer2001** [comment]
1. Sorry for the typo in my first response.
2. Cable connection is perfect (i.e. this mail was sent using the wired
connection)
3. I reinstalled - clean install- K. Dorn flavour of jamulus as I did not
know how to setup jamulus on an "official" Raspberry pi. Could It be simply
"apt-get install jamulus"? or something simple like that?
4. I am curious to learn how to enable the wifi communication on the fresh
installation. I launched -right click - the 2 arrows on the panel:
"Wireless & Wired network settings". It opened "Network Preferences" , but
wlan0 does not show in the interface selection. I also do not know where to
look for the driver.
Can you advise?
Thanks

**Reply by ann0see** [reply]
I think it's something like apt-get install jamulus but from the debian backports repository: https://packages.debian.org/buster-backports/jamulus

Unfortunately I don't know much about the wifi drivers, I found this which does the exact opposite. Maybe if you follow the steps you can also enable WiFi: https://pimylifeup.com/raspberry-pi-disable-wifi/

**Reply by ann0see** [reply]
Also see https://backports.debian.org/

**Comment by jfrommer2001** [comment]
Thanks for your  reply
I managed to enable and run the wifi communication on my Pi.
I'll gather detailed info and send it to you in a separate mail
Again, thank you very much

**Reply by ann0see** [reply]
Great to hear!

**Comment by jfrommer2001** [comment, marked as answer]
Further to my previous mail
in the section  [all] of  /boot/config.txt, I commented the line:
dtoverlay=disable-wifi
That's it
Thanks

**Reply by ann0see** [reply]
Ah. So it was even easier! Thank you.
